packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -111,4 +111,11 @@ Unlike normal disabled buttons, aria-disabled buttons can support tooltips.
 Aria-disabled buttons can operate as links, which also support tooltips.
 
 ```ts file="./ButtonAriaDisabledLinkTooltip.tsx"
+```
+
+### Buttons with a count label
+
+Buttons can now display a `count` in the form of a badge to indicate some value or number.
```

Comment:
We could omit "now" and just have the description start with "Buttons can display a count by passing in the `X` prop..."

The description will ultimately depend how the `count` and `isRead` props are handled based on previous feedback.